src/dymaptic.GeoBlazor.Core/Scripts/featureLayer.ts

```diff
@@ -75,14 +110,42 @@ export default class FeatureLayerWrapper {
         return await this.layer.queryObjectIds(jsQuery, options);
     }
 
-    async queryRelatedFeatures(query: DotNetRelationshipQuery, options: any, dotNetRef: any): Promise<FeatureSet | null> {
+    async queryRelatedFeatures(query: DotNetRelationshipQuery, options: any, dotNetRef: any, viewId: string | null)
```

Comment:
so the `queryFeatures` and the `queryRelatedFeatures` methods do essentially the same thing since they are almost identical with the primary difference being the use of the `graphicsDictionary` and `featureSetsDictionsary` in `queryRelatedFeatures` because we are following the featureSets relationships vs the features themselves?

Reply:
I don't have a ton of knowledge around this, I would defer to Esri's documentation or others on what the different queries mean.